### ISSUE TITLE
Baseline 1 smaller test set

### ISSUE THREAD

**Issue by MaksymDel** [mt2017-tartu-shared-task/nmt-system-D#2]
Please, take a smaller subset of your test set (e.g. 50.000 sentences) and compute BLEU on this (do not forget to undo preprocessing as described in lab3 text). Compare to the BLEU of 2.000.000 you already have and write a line about it in your report.

**Comment by papkov** [comment]
@maxdel related question: can we move parts of our big test- and dev-sets (2M and 1M) to train-set to improve training performance? Should we in this case retrain our baseline system?

**Comment by MaksymDel** [comment]
If you just move it to the train set and then make some modification on top of this, you will not know whether the change in result score comes from the additional data or from the modification. So if you decide to add these 3M training examples, you will have to retrain your baseline.

However, I highly do not recommend spending time (inc. computational) on this, and better concentrate on doing your planned modification!

 At the very end, before very final submission near the course end, you could just make a "final run" of your system with all the modifications and additional data included. I think it is the most optimal option if you really care about these sentences.

**Comment by papkov** [comment]
And if I simply take first 50k of dev-set without expanding train-set?  I think it can reduce computation time in our case. Would it also affect training process significantly?

**Comment by MaksymDel** [comment]
You reported devset of length 20.000 in your 2nd report and this is ok. You can take 50k as well if you want for some reason.

Just make sure you train your baseline and new (advanced with modification) systems on the same training set, validate on the same validation set, and report scores on the same test set for both systems.

**Comment by papkov** [comment]
Ok, thank you!
